src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -790,6 +790,7 @@ export default {
 				upcCreditRemaining: numeral(this.upcCreditRemaining).format('0.00'),
 				basketLoans: this.basketLoans,
 				promoName: this.campaignPartnerName,
+				removeLoanFromBasket: this.removeLoanFromBasket,
```

Comment:
Looks like this is defined as a function below. Prob don't need to dupe

Reply:
Looks like this is a prop wrapper for passing into a component in the template, so all of these props have the same setup. Looks to be by design.

Reply:
oh yea. you're right sorry. This isn't the `data` obj!